graphql/build.gradle

```diff
@@ -1,22 +1,22 @@
 plugins {
-    id "io.micronaut.build.internal.graphql-module"
+    id("io.micronaut.build.internal.graphql-module")
 }
 
 dependencies {
-    annotationProcessor mnSerde.micronaut.serde.processor
-    compileOnly mnSerde.micronaut.serde.support
+    annotationProcessor(mnSerde.micronaut.serde.processor)
+    compileOnly(mnSerde.micronaut.serde.api)
+    api(mn.jackson.annotations)
```

Comment:
can be compileOnly

Reply:
changed here: https://github.com/micronaut-projects/micronaut-graphql/pull/436/commits/0f36b4eab40f9f8ebce82e858448aed3d43cc105